frontend/src/metabase/visualizations/echarts/cartesian/option/grid.ts

```diff
@@ -1,43 +0,0 @@
-import type { GridOption } from "echarts/types/dist/shared";
```

Comment:
Most of the logic moved to https://github.com/metabase/metabase/pull/36879/files#diff-2225bf3348720497bacb168a0c058e6ab7058d598e51d175312a8dd7a06d69ffR155